### ISSUE TITLE
Error when using `vl_sift(image)`

### ISSUE THREAD

**Issue by norpadon** [IHPSystems/VLFeat.jl#14]
I am getting a following error when calling `vl_sift` on an image of type `Matrix{Gray{N0f8}}`:

```
ArgumentError: indexed assignment with a single value to possibly many locations is not supported; perhaps use broadcasting `.=` instead?

Stacktrace:
 [1] setindex_shape_check(::Float64, ::Int64)
   @ Base [./indices.jl:261](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/indices.jl:261)
 [2] _unsafe_setindex!(#unused#::IndexLinear, A::Matrix{Float64}, x::Float64, I::Base.LogicalIndex{Int64, BitMatrix})
   @ Base [./multidimensional.jl:925](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/multidimensional.jl:925)
 [3] _setindex!
   @ [./multidimensional.jl:916](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/multidimensional.jl:916) [inlined]
 [4] setindex!
   @ [./abstractarray.jl:1399](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/abstractarray.jl:1399) [inlined]
 [5] vl_sift(img::Matrix{Gray{N0f8}}; verbose::Bool, octaves::Int64, levels::Int64, firstOctave::Int64, edgeThreshold::Float64, peakThreshold::Float64, normThreshold::Float64, magnif::Float64, windowSize::Float64, floatDescriptors::Bool)
   @ VLFeat [~/.julia/packages/VLFeat/bMRDD/src/sift.jl:81](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/~/.julia/packages/VLFeat/bMRDD/src/sift.jl:81)
 [6] vl_sift
   @ [~/.julia/packages/VLFeat/bMRDD/src/sift.jl:1](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/~/.julia/packages/VLFeat/bMRDD/src/sift.jl:1) [inlined]
 [7] extract_features(image::Matrix{RGB{N0f8}})
   @ Main [~/Work/splatting/playground.ipynb:9](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/~/Work/splatting/playground.ipynb:9)
 [8] top-level scope
   @ [~/Work/splatting/playground.ipynb:1](https://file+.vscode-resource.vscode-cdn.net/Users/norpadon/Work/splatting/~/Work/splatting/playground.ipynb:1)
```

**Comment by stemann** [comment]
Thanks for the bug report - sounds like it wouldn’t be too hard to fix… (feel free to fix it and create a PR)

**Comment by stemann** [comment]
... I'll have a fix ready in a short while

**Comment by stemann** [comment]
Fixed with commit 162fb7a6f02f97ce4bf52f322a0e9349ec1c713a - available in v0.2.0 shortly: https://github.com/JuliaRegistries/General/pull/92379
